source/dubregistry/dbcontroller.d

```diff
@@ -509,9 +509,13 @@ struct DbRepository {
 		path.popFront;
 		if (path.empty || path.front.name.empty)
 			throw new Exception("Invalid Repository URL (missing project)");
-		project = path.front.name;
+
+		if(kind == "gitlab")
+			project = path.map!"a.name".join("/");
+		else
+			project = path.front.name;
 		path.popFront;
-		if (!path.empty)
+		if (!path.empty && kind != "gitlab")
```

Comment:
This would allow gitlab.com/foo/bar/foobar/fooobar/foo -  is this really possible?

edit: oh it looks like subgroups can be nested in subgroups

A comment about this in the source code would help future readers...

Reply:
I confirm, GitLab's subgroups are nestable. I'll add the comment as well

Reply:
Comment added.